lib/services/local/desktops.go

```diff
@@ -38,8 +38,61 @@ func NewWindowsDesktopService(backend backend.Backend) *WindowsDesktopService {
 	return &WindowsDesktopService{Backend: backend}
 }
 
+func (s *WindowsDesktopService) getWindowsDesktop(ctx context.Context, name, hostID string) (types.WindowsDesktop, error) {
+	key := backend.ExactKey(windowsDesktopsPrefix, hostID, name)
```

Comment:
Isn't this going to append a trailing slash? That's not how we're storing things, is it?
```suggestion
	key := backend.NewKey(windowsDesktopsPrefix, hostID, name)
```